Makefile

```diff
@@ -29,6 +30,13 @@ rebar:
 	chmod +x $@
 
 ui:
-	rm -rf apps/epl/priv/htdocs
-	yarn && yarn build
-	mv build apps/epl/priv/htdocs
+	@rm -rf apps/epl/priv/htdocs
+	@yarn && yarn build
+	@mv build apps/epl/priv/htdocs
+
+release:
+	@echo "Building release version of Erlang Performance Lab"
+	@make rebar
+	@make ui
+	@make
+	@./bootstrap
```

Comment:
I don't think so and am not sure how useful that `echo` is, especially when you can see `make release` right above